### DISCUSSION TITLE
Mute audio

### DISCUSSION THREAD

**Discussion by eskurisu** [cefsharp/CefSharp#4142] [Q&A] [answered]
I have multiple videos playing from cefsharp browser at winforms and I need to mute some of them at some point. Didn't find out how I can do this, now I am here to ask, would be appreciated a lot, if someone can help with it.

**Comment by amaitland** [comment, marked as answer]
Should look something like the following 

```c#
// add using CefSharp;
chromiumWebBrowser.GetBrowserHost().SetAudioMuted(true);
```

http://cefsharp.github.io/api/101.0.x/html/M_CefSharp_IBrowserHost_SetAudioMuted.htm
